.travis.yml

```diff
@@ -29,9 +29,10 @@ install:
   - pip install -r requirements_dev.txt
   - python setup.py install
   - make build
+  - export MPLBACKEND="agg"
 
 script:
-  - nosetests tests/base tests/tree tests/docs --with-cov --cov discretize --cov-config .coveragerc -v -s;
+  - nosetests tests/base tests/tree tests/docs tests/cyl --with-cov --cov discretize --cov-config .coveragerc -v -s;
```

Comment:
I parsed out the cyl tests so that they don't run on appveyor - they seem to cause memory issues (seems to be related to the import of matplotlib?) Travis is happy though :)